go/teams/list.go

```diff
@@ -281,7 +285,8 @@ func ListAll(ctx context.Context, g *libkb.GlobalContext, arg keybase1.TeamListT
 	}
 
 	tracer.Stage("Server")
-	teams, err := getTeamsListFromServer(ctx, g, "" /*uid*/, true /*all*/, false /* countMembers */)
+	teams, err := getTeamsListFromServer(ctx, g, "", /*uid*/
```

Comment:
change to `"" /*uid*/`

i.e. keep comment next to empty parameter

Reply:
hmm gofmt keeps changing this. should I change these back to be in one line?

Reply:
i guess so if we want to keep these comments...